recipes/openusd/all/conanfile.py

```diff
@@ -0,0 +1,490 @@
+        tc.variables["PXR_BUILD_TESTS"] = False
+        tc.variables["PXR_BUILD_EXAMPLES"] = False
+        tc.variables["PXR_BUILD_TUTORIALS"] = False
+        tc.variables["PXR_BUILD_HTML_DOCUMENTATION"] = False
+        tc.variables["PXR_ENABLE_PYTHON_SUPPORT"] = self.options.enable_python_support
+        tc.variables["PXR_BUILD_USD_TOOLS"] = self.options.build_usd_tools
+        
+        tc.variables["OPENSUBDIV_LIBRARIES"] = "opensubdiv::opensubdiv"
+        tc.variables["OPENSUBDIV_INCLUDE_DIR"] = self.dependencies['opensubdiv'].cpp_info.includedirs[0]
+        
+        tc.variables["TBB_tbb_LIBRARY"] = "TBB::tbb"
+        tc.variables["OPENVDB_LIBRARY"] = "OpenVDB::openvdb"
+        
+        tc.variables["PXR_ENABLE_MATERIALX_SUPPORT"] = self.options.enable_materialx_support
+        
+        tc.variables["PXR_BUILD_IMAGING"] = self.options.build_imaging
+        if self.options.build_imaging:
+            tc.variables["PXR_BUILD_IMAGING"] = True
+            tc.variables["PXR_ENABLE_GL_SUPPORT"] = self.options.enable_gl_support
+            tc.variables["PXR_ENABLE_VULKAN_SUPPORT"] = self.options.enable_vulkan_support
+            tc.variables["PXR_ENABLE_PTEX_SUPPORT"] = self.options.enable_ptex_support
+            tc.variables["PXR_ENABLE_OPENVDB_SUPPORT"] = self.options.enable_openvdb_support
+            tc.variables["PXR_BUILD_OPENIMAGEIO_PLUGIN"] = self.options.build_openimageio_plugin and self.options.build_gpu_support
+            tc.variables["PXR_BUILD_COLORIO_PLUGIN"] = self.options.build_opencolorio_plugin and self.options.enable_gl_support and self.options.build_gpu_support
+            tc.variables["PXR_BUILD_EMBREE_PLUGIN"] = self.options.build_embree_plugin
+            tc.variables["EMBREE_FOUND"] = self.options.build_embree_plugin
+            if self.options.build_usd_imaging:
+                tc.variables["PXR_BUILD_USD_IMAGING"] = True
+                if self.options.build_usdview:
+                    tc.variables["PXR_BUILD_USDVIEW"] = True
+
+        tc.variables["PXR_BUILD_PRMAN_PLUGIN"] = self.options.build_renderman_plugin
+        tc.variables["PXR_BUILD_ALEMBIC_PLUGIN"] = self.options.build_alembic_plugin
+        tc.variables["ALEMBIC_FOUND"] = self.options.build_alembic_plugin
+        tc.variables["PXR_ENABLE_HDF5_SUPPORT"] = self.options.build_alembic_plugin and self.options.enable_hdf5_support
+        tc.variables["PXR_BUILD_DRACO_PLUGIN"] = self.options.build_draco_plugin
+        tc.variables["PXR_ENABLE_OSL_SUPPORT"] = self.options.enable_osl_support
+        tc.variables["PXR_BUILD_ANIMX_TESTS"] = self.options.build_animx_tests
+
+        tc.generate()
+
+        tc = CMakeDeps(self)
+        tc.generate()
+
+        tc = VirtualBuildEnv(self)
+        tc.generate(scope="build")
+
+    def _patch_sources(self):
+        rmdir(self, os.path.join(self.source_folder, "cmake", "modules"))
+                              
+        if self.options.enable_materialx_support:
```

Comment:
As far as I understand, these are patches to use the target names that conan is producing so that the recipe consumes the same ones, righ?

Maybe patching could be avoided by modifying the `cmake_file_name`/`cmake_target_name` in the `CMakeDeps` generator for these dependencies?